CourierCore/Models/ConnectOptions.swift

```diff
@@ -19,6 +19,10 @@ public struct ConnectOptions: Equatable {
     public let userProperties: [String: String]?
     
     public let alpn: [String]?
+    
+    public let scheme: String?
```

Comment:
I probably need to update Courier Flutter client as well to pass the correct boolean flag for this based on the scheme from method channel. In case of courier iOS , scheme is not really used inside MQTTClientFramework hence i think its much more simpler to pass a single flag.

Reply:
Sorry mas. Is it better if we use `scheme` params? Because Courier Android and Courier Flutter use the scheme params too. Or maybe Courier Flutter will change it later for the parameters?

Reply:
agreed mas, my mistake. let's use scheme for consistency across platforms and remove that boolean flag